test/end-to-end/getRedirectionURL.test.js

```diff
@@ -377,5 +375,152 @@ describe("getRedirectionURL Tests", function () {
                 assert.equal(newUserCheck, "thirdpartypasswordless-true");
             });
         });
+
+        describe("No Redirection", function () {
```

Comment:
missing:
```
afterEach(function () {
                return screenshotOnFailure(this, browser);
            });
```

Reply:
It is present in the `after` function.

Reply:
The after function runs after all tests. Right now there is only one test in the describe block, but if someone adds another test, it may cause issues? Not sure.

Reply:
fixed it.